frontend/lib/src/components/widgets/Slider/Slider.test.tsx

```diff
@@ -392,16 +414,12 @@ describe("Slider widget", () => {
           "violet",
         ],
       })
-      const wrapper = mount(<Slider {...props} />)
-      const sliderDOMNodes = wrapper.find("div[role='slider']")
-      sliderDOMNodes.forEach(node => {
-        expect(node.getDOMNode().getAttribute("aria-valuetext")).toEqual(
-          "orange"
-        )
-      })
+      render(<Slider {...props} />)
+      const slider = screen.getByRole("slider")
+      expect(slider).toHaveAttribute("aria-valuetext", "orange")
     })
 
-    it("updates aria-valuetext correctly", () => {
+    it("updates aria-valuetext correctly", async () => {
```

Comment:
Curious question: Why does this test require `async`?

Reply:
Whoops - a leftover from experimenting with replicating user interactions - will remove